packages/cypress/cypress/integration/cucumber/step_definitions/refapp-3.x/12-patient-conditions/patient-conditions.js

```diff
@@ -0,0 +1,40 @@
+import { Given, Before, When } from "cypress-cucumber-preprocessor/steps";
+
+let identifier = null;
+let patient = null;
+
+Before({ tags: "@patient-conditions" }, () => {
+  cy.generateIdentifier().then((generatedIdentifier) => {
+    identifier = generatedIdentifier;
+    cy.createPatient(identifier).then((generatedPatient) => {
+      patient = generatedPatient;
+    });
+  });
+});
+
+Given("the user is logged in", () => {
+  cy.login();
+});
+
+Given("the user arrives on a patient’s summary page", () => {
+  cy.visit(`patient/${patient.uuid}/chart`);
+});
+
+When("the user clicks on Conditions tab", () => {
+  cy.contains("Conditions").click({ force: true });
```

Comment:
Any specific reasons to force click?

Reply:
The comment applies to other force clicks too.

Reply:
We can use the `click()`method without force click. But randomly cypress won't click the tabs in the patient chart for some reason. @jayasanka-sack